src/workerd/api/tests/compat-flags-test.js

```diff
@@ -0,0 +1,23 @@
+import {
+  ok,
+  throws,
+} from 'node:assert';
+
+import { compatFlags } from 'cloudflare:workers';
```

Comment:
for consistency with wrangler.toml, and capnp, I think we should call this symbol `compatibilityFlags`.